frontend/model/contracts/shared/constants.js

```diff
@@ -24,7 +24,8 @@ export const GROUP_DESCRIPTION_MAX_CHAR = 500
 export const GROUP_PAYMENT_METHOD_MAX_CHAR = 250
 export const GROUP_NON_MONETARY_CONTRIBUTION_MAX_CHAR = 150
 export const GROUP_CURRENCY_MAX_CHAR = 10
-
+export const GROUP_MAX_PLEDGE_AMOUNT = 1000000000
+export const GROUP_MAX_PLEDGE_AMOUNT_STRING = '1,000,000,000$'
```

Comment:
The `$` should be in front. But nevermind that, this constant shouldn't be here at all, please use the getter `withGroupCurrency` instead.